frontend/src/component/changeRequest/ChangeRequest/Changes/Change/ChangeActions.tsx

```diff
@@ -149,25 +152,56 @@ export const ChangeActions: FC<{
                                                 Edit change
                                             </Typography>
                                         </ListItemText>
-                                        <EditChange
-                                            changeRequestId={changeRequest.id}
-                                            featureId={feature}
-                                            change={
-                                                change as
-                                                    | IChangeRequestAddStrategy
-                                                    | IChangeRequestUpdateStrategy
+                                        <ConditionallyRender
+                                            condition={newStrategyConfiguration}
+                                            show={
+                                                <NewEditChange
```

Comment:
Can we extract those? With so many nesting levels it gets difficult to read as the code becomes very narrow

Reply:
What do you refer to when you say "those?" Do you mean to have the components in variables and use those instead?

Reply:
anything that will remove whitespace on the left:
<img width="488" alt="Screenshot 2023-12-15 at 14 06 47" src="https://github.com/Unleash/unleash/assets/1394682/45ea6402-aa1b-4639-8632-b47df9dec83e">
It's mostly those deeply nested components that get shifted to the right

Reply:
can be done in followup PR

Reply:
Ok, thanks! I'll see what I can do, there is some cleanup to be done here.